build-customization.rst

```diff
@@ -0,0 +1,72 @@
+.. post:: Nov 10, 2022
+   :tags: announcement, feature, builds
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+      We released user-defined commands (``build.commands`` config key)
+      which allow users to overwrite the build process completely.
+
+
+Overwrite the build process completely
+======================================
+
+We are happy to announce a new *beta* feature that allow users to **overwrite the build process completely**.
+In the past, `we talked about executing custom commands <https://blog.readthedocs.com/user-defined-build-jobs/>`_ *in-between* the Read the Docs build process.
+However, that approach may not be enough for some projects with strong custom build processes
+and they were not able to use our platform because of that.
```

Comment:
```suggestion
However, that approach isn't sufficient for projects with a heavily customized build processes,
some of which are not able to use our platform at all.
```

Reply:
I like your suggestion, but I would keep the "may" since there are projects with heavy customized builds where `build.jobs` is enough for them.

What about the following?

 ```suggestion
However, that approach may not be sufficient for some projects with a heavily customized build process,
some of which are not able to use our platform at all.
```

👍🏼 or 👎🏼 ?

Reply:
:+1: or actually I just committed your suggestion :)